assets/sass/v2/_footer.scss

```diff
@@ -1,7 +1,11 @@
 .siw-main-footer {
   .auth-footer {
+    .link {
+      margin-right: 10px;
```

Comment:
I wonder it might be better to align links vertically. (maybe another JIRA?)

<img width="355" alt="Screen Shot 2019-08-16 at 3 57 39 PM" style="border:1px solid black;" src="https://user-images.githubusercontent.com/13139863/63202464-a4637e80-c03e-11e9-896c-55d24a9da31b.png">

Reply:
Yup, agree. We are running out of space